src/components/FilterLink/FilterLink.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { FilterType } from '../../types';
+
+interface FilterLinkProps {
+  type: FilterType;
+  filterType: FilterType;
+  onChangeType: (type: FilterType) => void;
+}
+
+export const FilterLink: React.FC<FilterLinkProps> = ({
+  type,
+  filterType,
+  onChangeType,
+}) => {
+  const handleClick = () => {
+    onChangeType(type);
+  };
+
+  return (
+    <a
+      href={`#/${type.toLowerCase()}`}
+      className={cn('filter__link', {
+        selected: filterType === type,
+      })}
+      data-cy={`FilterLink${type}`}
+      onClick={handleClick}
+    >
+      {type.charAt(0).toUpperCase() + type.slice(1).toLowerCase()}
```

Comment:
why you need this. 
  ```
ALL = 'All',
  ACTIVE = 'Active',
  COMPLETED = 'Completed',
```
value in your enum, looks already correct. Or am I wrong?

Reply:
Oh, i changed this, o don`t why this not save. I am write git add ./src it is must be changed. BTW just push this change next time also